cumulus/client/consensus/aura/Cargo.toml

```diff
@@ -59,6 +59,11 @@ polkadot-node-subsystem-util = { workspace = true, default-features = true }
 polkadot-overseer = { workspace = true, default-features = true }
 polkadot-primitives = { workspace = true, default-features = true }
 
+[dev-dependencies]
+cumulus-test-client.workspace = true
+sp-keyring.workspace = true
+cumulus-test-relay-sproof-builder.workspace = true
```

Comment:
While I like this syntax as well, we want to stay compatible with the rest of the code base.